docs/tutorials/drm-config.md

```diff
@@ -145,18 +145,41 @@ playback.  Passing in a higher security level than can be supported will cause
 default is the empty string, which is the lowest security level supported by the
 key system.
 
-Each key system has their own values for robustness.  The values for Widevine
-are well-known (see the [Chromium sources][]) and listed below, but
-values for other key systems are not known to us at this time.
+Each key system has their own values for robustness.
 
-[Chromium sources]: https://cs.chromium.org/chromium/src/components/cdm/renderer/widevine_key_system_properties.h?q=SW_SECURE_CRYPTO&l=22
+##### Widevine
+
+Chromium sources: https://cs.chromium.org/chromium/src/components/cdm/renderer/widevine_key_system_properties.h?q=SW_SECURE_CRYPTO&l=22
 
 - `SW_SECURE_CRYPTO`
 - `SW_SECURE_DECODE`
 - `HW_SECURE_CRYPTO`
 - `HW_SECURE_DECODE`
 - `HW_SECURE_ALL`
 
+##### PlayReady
+
+Microsoft Documentation: https://docs.microsoft.com/en-us/playready/overview/security-level
+
+- `3000`
+- `2000`
+
+`com.microsoft.playready` key system ignores given robustness and stays at a
+`2000` decryption level.
+
+If you specify a robustness into the configuration, Shaka will try to load and
+start the key session with the `com.microsoft.playready.recommendation` key
+system. That key system correctly implements the EME specification and will use
+the given decryption robustness.
+
+It's up to you to detect and set the robustness if the playing platform is able
+to handle it.
```

Comment:
I would prefer you do this differently.

Instead of forcing `recommendation` based on robustness, but requiring the app to know if the platform can handle the requested robustness level, why not try `recommendation` unconditionally, then fall back to the old key system ID?

That way, one config can be used for both desktop Edge and embedded devices with PlayReady under the old ID.  It would also be a nice to take advantage of PR bug fixes in the new key system ID, even when robustness isn't set by the app.

Reply:
👋 @joeyparrish 

Thank you for taking the time to review !

I didn't want to go that way, as @OrenMe cited, there is no good way of doing this without adding real complexity with a test & fallback logic only for PlayReady where recommendation is not supported. Where would the check be ? When filling in the DRM information, when actually trying to request the media key system access ?

I think that asking not to set robustness to get the legacy PlayReady key system access matches with its "implementation": `com.microsoft.playready ignores robustness completely`. If you set the robustness, you actually need to ensure that robustness is given back to the CDM (even if it's the default `2000`) and prevent false-positives playbacks with the non-EME compliant `com.microsoft.playready` keySystem.

If you think that's the good way to land the feature, I'll develop it that way :)

Reply:
Hi @joeyparrish,

Any thoughts about my comment above ? Thank you !

Reply:
@TheModMaker Works for me then :)

Reply:
Done !